Cubical/Foundations/Isomorphism.agda

```diff
@@ -174,12 +174,24 @@ infix   1 _∎Iso
 codomainIsoDep : ∀ {ℓ ℓ' ℓ''} {A : Type ℓ} {B : A → Type ℓ'} {C : A → Type ℓ''}
                  → ((a : A) → Iso (B a) (C a))
                  → Iso ((a : A) → B a) ((a : A) → C a)
-Iso.fun (codomainIsoDep is) f a = Iso.fun (is a) (f a)
-Iso.inv (codomainIsoDep is) f a = Iso.inv (is a) (f a)
-Iso.rightInv (codomainIsoDep is) f = funExt λ a → Iso.rightInv (is a) (f a)
-Iso.leftInv (codomainIsoDep is) f = funExt λ a → Iso.leftInv (is a) (f a)
+fun (codomainIsoDep is) f a = fun (is a) (f a)
+inv (codomainIsoDep is) f a = inv (is a) (f a)
+rightInv (codomainIsoDep is) f = funExt λ a → rightInv (is a) (f a)
+leftInv (codomainIsoDep is) f = funExt λ a → leftInv (is a) (f a)
 
 codomainIso : ∀ {ℓ ℓ' ℓ''} {A : Type ℓ} {B : Type ℓ'} {C : Type ℓ''}
            → Iso B C
            → Iso (A → B) (A → C)
 codomainIso z = codomainIsoDep λ _ → z
+
+
+Iso≡Set : isSet A → isSet B → (f g : Iso A B)
```

Comment:
@ecavallo Is this somewhere else in the library?

Reply:
Not to my knowledge.

Reply:
Is one-side homotopy enough? Because the inverse function is unique.

Reply:
True! You could also get away with only asking that one type is a set, since it follows automatically that the other is (but I suppose this could lead to bigger terms).